### PR TITLE
[Bug fix]: Fixed Heroku DB Configuration

### PR DESCRIPTION
**Pull request by ibutiti**
## PR Status
**READY**

## Story ID
[#154015704](https://www.pivotaltracker.com/story/show/154015704)

## Added Migrations
NO

## Description
Heroku DB settings were not correctly picked up by django in the settings.py file. Django would default to sqlite instead of configuring postgres. Added a local_settings.py file with the new config. Environment variable (config variables as Heroku calls them) `HEROKU_DEPLOYMENT` in Heroku app should be set to 1 for this to work.

## New Packages Added
None

## Related PRs 
Fixes for PR2 below:

branch | PR
------ | ------
ibutiti:master | [Heroku Deployment Changes](https://github.com/andela/hc-anansi/pull/2)

## Todos
- [x] Tested and working locally
- [x] Tested and working on heroku
- [ ] Reviewed and approved by a TTL
- [ ] Deployment to team Heroku account

## Impacted Areas in the application
`hc/settings.py`: removed heroku db configs from this file
`hc/local_settings.py`: Added heroku db configs to this file

### CODE REVIEW

File: hc/.gitignore [hunk 1]
```diff
@@ -0,0 +1,103 @@
+# Byte-compiled / optimized / DLL files
```
Comment (johnseremba) [inline hc/.gitignore]: Why did you have to place the `.gitignore` under the `hc/` directory??
I guess it is supposed to be kept in the root folder.

File: hc/local_settings.py [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import os
+import dj_database_url
+
+# If deploying to heroku, the database config will be populated automatically
+# Create an environment variable HEROKU_DEPLOYMENT in heroku and set it to 1
+if os.environ.get("HEROKU_DEPLOYMENT") == '1':
+    DATABASES = {
```
Comment (johnseremba) [inline hc/local_settings.py]: I guess this can be done in the `hc/settings.py` file. You just need to reference it correctly.

**Review by johnseremba — CHANGES_REQUESTED** [review]

File: hc/.gitignore (hunk 1 above)
Comment (ibutiti) [reply hc/.gitignore]: My bad, I may have dragged the file into `hc/` unknowingly. Will sort it out

File: hc/local_settings.py (hunk 2 above)
Comment (ibutiti) [reply hc/local_settings.py]: Noted and will resolve.

**Comment by ibutiti** [comment]
Requested changes done

File: hc/.gitignore (hunk 1 above)
Comment (johnseremba) [reply hc/.gitignore]: Not yet sorted

**Review by johnseremba — APPROVED** [review]
looks good to me

**Pull request merged**